### ISSUE TITLE
Search not updating as field is filled

### ISSUE THREAD

**Issue by mrtenda** [itchio/itch#2379]
When I search for games by title in the app, they often don't come up. However, the search on the itch.io website seems to work fine. This is making it hard for me to use the app to find specific games. I tested this on both my desktop and my laptop (both Windows 10) and see the same behavior.

For example, I just opened Itch.io in my browser, and the featured game was "Biomass":

![image](https://user-images.githubusercontent.com/1281326/79052048-3ac65f00-7c23-11ea-937d-1be94b3a60b2.png)

If I open the Itch app and type "biomass" in the search bar, this is what I see:

![image](https://user-images.githubusercontent.com/1281326/79052077-69443a00-7c23-11ea-89e0-36c0348a9d2d.png)

Interestingly, if I copy and past the word "biomass" into the search bar instead of typing it, the problem seems not to occur and I get this result instead. (I suspect this might have to do with the search query happening when I type the letter "b" in "biomass" but not updating after the entire word is typed)

![image](https://user-images.githubusercontent.com/1281326/79052155-0a32f500-7c24-11ea-81ef-ae510ab1b62b.png)

Also, for reference, if I search for "biomass" on itch.io in my browser, this is what I see:

![image](https://user-images.githubusercontent.com/1281326/79052119-a9a3b800-7c23-11ea-87e2-bfb6e14c9a48.png)

(As a side note, it seems strange to me that the Biomass game isn't the first search result on the itch.io website when I search for "biomass", but I guess that would be a different issue that isn't related to the app)

Thanks for developing itch and looking at my issue.

OS: Windows 10
itch version: 25.4.0.

**Comment by fasterthanlime** [comment]
Yep, definitely an issue I've tried to avoid when designing it but have experienced in the stable build a few times. Thanks for the detailed report!

**Comment by MarjaE2** [comment]
Searches by publisher aren't working either.

I also have a very hard time trying to navigate the list, because it flashes and expands or shrinks listings, because the font is small, while-on-black, and hard to read, and because it disappears.

**Comment by leafo** [comment]
Search has been rebuilt and will launch in next version of the app.

* The autocomplete now only shows local results, things fetched from your collections, purchases, bundles, etc.
* You can now select "Search on itch.io..." and it will open up the full results page on the site for the query
